src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/ObTableAggregationResult.java

```diff
@@ -43,11 +43,7 @@ public ObTableAggregationResult(QueryResultSet queryResultSet, Map<Integer, Stri
      */
     public void init() throws Exception {
         this.queryResultSet.next();
-        List<ObObj> init_row;
-        init_row = this.queryResultSet.get_Row();
-        for (int i = 0; i < init_row.size(); i++) {
-            row.put(message.get(i), init_row.get(i).getValue());
-        }
+        row = this.queryResultSet.getRow();
```

Comment:
Have you ever tested ObTableAggregationResult on a non-existent raw or an empty raw?

Reply:
fixed